### PR TITLE
Modified Japanese translation

### PR DESCRIPTION
**Pull request by koji**
RE: #1638

I have verified that this pull request:

* [x] has no linting errors (`npm run lint`)
* [x] is from a uniquely-named feature branch and has been rebased on top of the latest `develop` branch. (If I was asked to make more changes, I have made sure to rebase onto `develop` then too)
* [x] is descriptively named and links to an issue number, i.e. `Fixes #123`

### CODE REVIEW

**Comment by koji** [comment]
@micuat
@deconbatch
@takawo
@sugi2000

Updated items(JSON file) that were pointed out by Naoto HIÉDA san. The following link still has some comments on the translation. I would like to hear your thought.
Thanks!


Naoto HIÉDAさんから頂いた指摘を反映させました。
下記のリンクのコメントについて、他の方の意見を伺えればと思います。
よろしくお願いします。

https://github.com/processing/p5.js-web-editor/pull/1664/files

特に、共有 or シェアについて
Facebook, Instagram: シェア
Google, Youtube: 共有

**Comment by sugi2000** [comment]
@koji 日本語化すばらしいです。
・「共有」が一般的かなとおもいます。
・「下さい」は「ください」に統一してはいかがでしょうか。

**Comment by catarak** [comment]
Yay! Just remove the draft status from this PR when it's ready to be merged 😄

File: translations/locales/ja/translations.json
Comment (micuat) [inline translations/locales/ja/translations.json]: コレクション「が」ありません
なのでどちらかに統一したほうがよさそうです

File: translations/locales/ja/translations.json
Comment (micuat) [inline translations/locales/ja/translations.json]: もしくは

File: translations/locales/ja/translations.json
Comment (micuat) [inline translations/locales/ja/translations.json]: > ログイン もしくは アカウント作成.

と表示されるので、「アカウントを作成する」のほうがきれいかもしれません

Comment (micuat) [reply translations/locales/ja/translations.json]: ちなみにこんな感じ

image

それとは別に、メニューの方は「アカウント作成」のままでいいと思います（あるいは「アカウントを作成」）

image

**Review by micuat — CHANGES_REQUESTED** [review]
細かい点など、docker で実際に使用してみての感想です

**Comment by micuat** [comment]
あと、ここは「リファレンス（英語）」としたほうがいいかも
https://github.com/processing/p5.js-web-editor/blob/df6cbff81f5a8b0a914b82c146c31bb326a4cbd5/translations/locales/ja/translations.json#L31

**Comment by catarak** [comment]
It's hard for me to tell if this is ready to be merged or there's still a few remaining comments from @micuat. @koji let me know?

**Comment by koji** [comment]
@catarak 
I need to update the JSON file to reflect his feedback. Once everything is done, I will let you know.
Thanks!

**Comment by micuat** [comment]
@catarak thanks, I think I have a member permission so once it's ready I can "approve" the pull request then you can merge it - does that sound good?

File: translations/locales/ja/translations.json
Comment (koji) [reply translations/locales/ja/translations.json]: ありがとうございます。

**Comment by koji** [comment]
> リファレンス（英語）
ああ、確かにそっちの方が親切ですね。

File: translations/locales/ja/translations.json
Comment (koji) [reply translations/locales/ja/translations.json]: `アカウントを作成してください`とかの方がUIの文言としては自然ですかね？

**Comment by micuat** [comment]
丁寧語で統一するのいいアイディアだと思います！

Looks good, should we wait for more comments or merge now? Unless there is an obvious fix, I think we should finish reviewing and if there is something to fix we can open a new issue/pr.

**Comment by koji** [comment]
I think we can go forward.
Yeah, I agree with you. We can open a new issue/pr to fix an issue.
Anyway, thank you for your help. Really appreciate🙏

**Review by micuat — APPROVED** [review]
finished revision! @catarak please merge when you can :)

**Review by micuat — APPROVED** [review]
(i'm retrying because i might have submitted review on the wrong commit)

...so obviously I don't have permission to approve 🤣

**Review by catarak — APPROVED** [review]

**Comment by catarak** [comment]
yay! so excited this translation is done. when i deploy this i'm gonna post about it on the p5.js twitter 😄

**Pull request merged**
